generator/config/accumulator/count.yaml

```
@@ -10,28 +10,64 @@ description: |
     Distinct from the $count pipeline stage.
     New in MongoDB 5.0.
 tests:
-    -
-        name: 'Use in $group Stage'
-        link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/count-accumulator/#use-in--group-stage'
-        pipeline:
-            -
-                $group:
-                    _id: '$state'
+    - name: Use in $group Stage
+      link: https://www.mongodb.com/docs/manual/reference/operator/aggregation/count-accumulator/#use-in--group-stage
+      pipeline:
+          - $group:
+                _id: $state
+                countNumberOfDocumentsForState:
+                    $count: {}
+      schema:
+          cakeSales:
+              _id:
+                  types:
+                      - bsonType: Number
```

Should we use a less verbose syntax?
```suggestion
                      - Number
```

Is this Camelcase name a cross-driver convention? I used lowercase: https://github.com/mongodb/mongo-php-library/blob/75fa63ea5c72e735efbc005ea5b6de8df7c45ec0/generator/config/expressions.php#L19-L38

---

I believe the format here was chosen to match the format in the [mongodb-schema package](https://github.com/mongodb-js/mongodb-schema), while the casing there is likely intended to match the documentation: https://www.mongodb.com/docs/manual/reference/bson-types/.